vllm/model_executor/layers/attention.py

```diff
@@ -357,11 +357,12 @@ def set_attn_bias(self, input_metadata: InputMetadata) -> None:
             # be sliced from a tensor whose length is a multiple of 8.
             padded_len = (prompt_len + 7) // 8 * 8
             bias = torch.empty(
+                1,  # batch_size
                 self.num_heads,
-                padded_len,
+                prompt_len,
```

Comment:
What is the reason for the change from `padded_len` to `prompt_len` here?

Reply:
Good question. This is actually a right implementation to fulfill the alignment requirement in xformers. This will fix #795